### ISSUE TITLE
Deploy a multisig on testnet and direct the mining rewards there

### ISSUE THREAD

**Issue by axic** [ewasm/testnet#120]

**Comment by jwasinger** [comment]
Suggesting we close this or move it to a later milestone.

**Comment by axic** [comment]
Is deploying a multisig such a challenge?

**Comment by jwasinger** [comment]
No.  But if we wipe the testnet (which happens regularly to staging) then we need to redeploy.

What's the use-case for this multisig?

**Comment by axic** [comment]
Is it not possible to deploy it as part of ansible?

**Comment by axic** [comment]
We need a multisig for the demo governance stuff too.

**Comment by jwasinger** [comment]
what's a good multi-sig to use for this?

**Comment by axic** [comment]
The "gnosis multisig" seems to be the best one out there: https://github.com/gnosis/MultiSigWallet

`truffle migrate <account1,account2,...,accountN> <requiredConfirmations>`

**Comment by axic** [comment]
Would also be nice deploying the dapp (https://github.com/gnosis/MultiSigWallet/tree/master/dapp), though not necessary because https://wallet.gnosis.pm can be set to an RPC.

**Comment by hugo-dc** [comment]
Multisig Wallet can also be deployed using the web interface: https://wallet.gnosis.pm

**Comment by axic** [comment]
What is the address?

**Comment by hugo-dc** [comment]
@axic 0x697c7a5e69fb6bd75eb2e0763d526f1f2fddeb43
